components/series/stackedBar.js

```diff
@@ -88,8 +52,12 @@
 
         stackedBar.yScale = fc.utilities.property(d3.scale.linear());
 
-        stackedBar.xValueKey = fc.utilities.property('name');
+        // Implicitly dependant on the implementation of the stack layout's `out`.
+        stackedBar.y0 = fc.utilities.property(function(d) {
+            return d.y0;
+        });
 
-        return stackedBar;
+        var stackLayout = d3.layout.stack();
```

Comment:
I can see how this could be confusing so I'll change that. I was aware of variable hoisting but I didn't consider that the assignment wasn't - that's good to know.